python/pipeline/reso.py

```diff
@@ -1065,13 +1065,20 @@ def _make_tuples(self, key):
             MaskClassification.Type().insert1({**key, 'mask_id': mask_id, 'type': mask_type})
 
 
+
+
 @schema
 class ScanSet(dj.Computed):
     definition = """ # set of all units in the same scan
-
     -> Fluorescence                 # processing done per slice
     """
 
+    def _job_key(self, key):
```

Comment:
Will this be called job_key or _job_key, because we already have job_key below. I don't think any of them work now so we can delete it later